src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -51,6 +51,11 @@ protected Invitation()
                 throw new ArgumentNullException(nameof(project));
             }
 
+            if (project.Plant != plant)
+            {
+                throw new ArgumentException($"Plant {plant} should be equal to project plant {project.Plant}");
```

Comment:
throw new ArgumentException($"Can't relate {nameof(project)} in {project.Plant} to item in {plant}");